client/index.jade

```diff
@@ -85,8 +85,8 @@ html
         div(class="container")
             div(class="panel panel-default")
                 div(class="panel-body")
-                    p(class="text-info") Don't forget to tell 
-                        a(href="https://twitter.com/intent/tweet?screen_name=wardleymaps" target="_blank") @wardleymaps 
+                    p(class="text-info") Don't forget to tell
```

Comment:
Spaces in this and next line are important.
![7582620](https://cloud.githubusercontent.com/assets/1265262/6430746/8e410298-c017-11e4-86ac-557cb5c759fc.png)